app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        mv, current_file, copied_file_path = command.split()
```

Comment:
1. mv is not the correct naming, it is 'command'
2. You need to validate the command. What happens if there are more than 3 values?